lib/tcpip/tcpip_error_handler.h

```diff
@@ -31,24 +31,20 @@ extern struct FlagLockState ipThreadLockState;
 /**
  * Other locks and futexes to release as part of the reset. These are
  * documented in `sdk/include/FreeRTOS-Compat/task.h` in the main tree
- * (`__CriticalSectionFlagLock` and `__SuspendFlagLock`) and
- * `source/FreeRTOS_IP.c` in the FreeRTOS+TCP tree (`xNetworkEventQueue`).
+ * (`__CriticalSectionFlagLock` and `__SuspendFlagLock`),
+ * `source/FreeRTOS_IP.c` in the FreeRTOS+TCP tree (`xNetworkEventQueue`), and
+ * `network_wrapper.cc` in the TCP/IP wrapper.
  */
 extern struct RecursiveMutexState __CriticalSectionFlagLock;
 extern struct RecursiveMutexState __SuspendFlagLock;
 extern QueueHandle_t              xNetworkEventQueue;
+extern FlagLockPriorityInherited  sealedSocketsListLock;
```

Comment:
Missing doc comment.

Reply:
Actually these four locks are documented with the one big doc comment above them, which I updated accordingly in the PR?